calico_upgrade/pkg/commands/dryrun.go

```diff
@@ -47,28 +47,28 @@ Options:
                                configuration in YAML or JSON format for
                                the Calico v3 API.
                                [default: ` + constants.DefaultConfigPathV1 + `]
-  --output-dir=<OUTPUTDIR>     Directory in which the data migration reports
-                               are written to.
+  --output-dir=<OUTPUTDIR>     Directory to store the data migration reports in.
                                [default: ` + constants.GetDefaultOutputDir() + `]
   --ignore-v3-data             Ignore any existing Calico data that is in the
-                               v3 format.  If there is v3 data present, we
+                               v3 format. If there is v3 data present, we
                                recommend you remove all Calico data from the
                                v3 datastore before upgrading, however, this
                                option may be used if that is not possible, or
                                if you know all of the data present will be
                                updated by the upgrade.
 
 Description:
-  Validate that the Calico v1 format data can be migrated to Calico v3 format
-  required by Calico v3.0+.
+  This command performs a dryrun of the data migration. This validate that
+  the v1 formatted data will be successfully converted and that the v3
```

Comment:
Its likely emily would do away with "will be successfully ..." and replace with some pithy present tense construct, but I can't think what it might be :-) probably punt for now.